src/Samples/Sample.AspNetCore.SystemTests/Test/Base/TestBase.cs

```diff
@@ -1,92 +1,79 @@
-        public void GlobalSetup()
-        {
-#if DEBUG
-            AtataContext.GlobalConfiguration.
-                UseChrome().
-                    WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.Chrome) as ChromeOptions).
-                UseFirefox().
-                    WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.Firefox) as FirefoxOptions).
-                UseInternetExplorer().
-                    WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.InternetExplorer) as InternetExplorerOptions).
-                AddNUnitTestContextLogging().
-                WithMinLevel(LogLevel.Trace).
-                TakeScreenshotOnNUnitError().
-                    AddScreenshotFileSaving().
-                        WithFolderPath(() => $@"Logs\{AtataContext.BuildStart:yyyy-MM-dd HH_mm_ss}").
-                        WithFileName(screenshotInfo => $"{AtataContext.Current.TestName} - {screenshotInfo.PageObjectFullName}").
-                UseTestName(() => $"[{_driverAlias}]{TestContext.CurrentContext.Test.Name}");
-#endif
-        }
-
-
-        [SetUp]
-        public void SetUp()
-        {
-#if DEBUG
-            AtataContext.Configure()
-                .UseDriver(_driverAlias)
-                    .UseBaseUrl("https://localhost:44344/")
-            .Build();
-            AtataContext.Current.Driver.Maximize();
-#elif DEV
-            AtataContext.Configure()
-                .UseDriver(_driverAlias)
-                    .UseBaseUrl("https://YourBaseUrl.com/")
-            .Build();
-            AtataContext.Current.Driver.Maximize();
-#elif RELEASE
-
-            AtataContext.Configure()
-                .UseChrome()
-                .WithOptions(DriverOptionsFactory.GetDriverOptions(Driver.Chrome) as ChromeOptions)
-                .UseBaseUrl(Environment.GetEnvironmentVariable("Swedbank.Pay.Sdk.SampleWebsite.BaseUrl", EnvironmentVariableTarget.User))
-                .Build();
-            AtataContext.Current.Driver.Maximize();
-#endif
-        }
-
-        protected TestBase(string driverAlias) => this._driverAlias = driverAlias;
-
-        [TearDown]
-        public void TearDown()
-        {
-            AtataContext.Current?.CleanUp();
-        }
-
-
-        [OneTimeTearDown]
-        public void GlobalDown()
-        {
-            foreach (Driver driverType in Enum.GetValues(typeof(Driver)))
-                WebDriverCleanerService.KillWebDriverProcess(WebDriverCleanerService.DriverNames[driverType]);
-        }
-    }
+﻿using System;
+using Atata;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.IE;
+
+using Sample.AspNetCore.SystemTests.Services;
+
+namespace Sample.AspNetCore.SystemTests.Test.Base
+{
+    using static Drivers;
+
+    [TestFixture(DriverAliases.Chrome)]
+    public abstract class TestBase
+    {
+        private readonly string _driverAlias;

```

Comment:
Codacy found an issue: [Remove this unread private field '_driverAlias' or refactor the code to use its value.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5523315)